include/openPMD/IO/IOTask.hpp

```diff
@@ -65,8 +68,10 @@ OPENPMDAPI_EXPORT_ENUM_CLASS(Operation)
     DELETE_ATT,
     WRITE_ATT,
     READ_ATT,
-    LIST_ATTS
-};  //Operation
+    LIST_ATTS,
+
+    ADVANCE
```

Comment:
We missed the opportunity to document the operations, but let's start with the new ones:
```suggestion
    LIST_ATTS,

    ADVANCE //!< ...
```

If it's too brief here, then let's add it to the specializations below.